OCRManager/Impl/OcrParser.cs

```diff
@@ -1,98 +1,43 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace OCRManager.Impl
 {
     public class OcrParser : IOcrParser
     {
-        private const int _lineLength = 27;
-
         public string UnknownDigit => "?";
 
         public string ParseInput(string input)
         {
-            IEnumerable<string> parsedStrings = InternalParseInput(input);
+            var parsedStrings = InternalParseInput(input);
 
             return string.Concat(parsedStrings.Select(DigitHelper.GetDigitString)).Replace(DigitHelper.UnknownDigitString, UnknownDigit);
         }
 
         public string ExtractIllFormedDigit(string input)
         {
-            IEnumerable<string> parsedStrings = InternalParseInput(input);
+            var parsedStrings = InternalParseInput(input);
 
             return parsedStrings.Single(str => DigitHelper.GetDigitString(str).Contains(DigitHelper.UnknownDigitString));
         }
 
-        private IEnumerable<string> InternalParseInput(string input)
-        {
-            ICollection<StringBuilder> digits = new List<StringBuilder>(9)
-            {
-                new StringBuilder(_lineLength), new StringBuilder(_lineLength), new StringBuilder(_lineLength),
-                new StringBuilder(_lineLength), new StringBuilder(_lineLength), new StringBuilder(_lineLength),
-                new StringBuilder(_lineLength), new StringBuilder(_lineLength), new StringBuilder(_lineLength)
-            };
-
-            var inputInLines = input.Replace("\r", string.Empty).Split('\n');
-
-            for (int lineIndex = 1; lineIndex < inputInLines.Length; ++lineIndex)
-            {
-                if (inputInLines[lineIndex]?.Length == 0)
-                {
-                    inputInLines[lineIndex] = new string(' ', _lineLength);
```

Comment:
Tests are failing due to the removal of this operation.
Either this part needs to be re-added or tests need to be modified (adding a bunch of spaces to the first line of failing datasets, `111111111` and `444444444`).

Reply:
annoying, these all passed locally!